Tribler/community/doubleentry/database.py

```diff
@@ -0,0 +1,266 @@
+""" This file contains everything related to persistence for DoubleEntry.
+"""
+import base64
+
+from os import path
+
+from Tribler.dispersy.database import Database
+
+
+def encode_db(db_object):
```

Comment:
Do you object to the use of mapping this function to the list of data to be inserted into the database? I could transform it manually at the point where I have that list.

Or do you object to the laziness of the inner workings of the method?
I can change it so that it checks the type and handles more accordingly for binary.